modules/social_features/social_comment/social_comment.install

```diff
@@ -83,3 +83,21 @@ function social_comment_update_8001(&$sandbox) {
     }
   }
 }
+
+/**
+ * Enable 'delete all comments' permission for site manager users.
+ */
+function social_comment_update_8002(&$sandbox) {
```

Comment:
@vladnosykov Please add this also to `hook_instal()` so we will have this permission by default in new installations.